charts/spiffe-helper-csi-driver/values.yaml

```diff
@@ -0,0 +1,22 @@
+## @skip global
+global: {}
+
+## @param spiffeHelper.image.registry The OCI registry to pull the image from
```

Comment:
Might want to clarify "spiffe helper image"

Reply:
```suggestion
## @param spiffeHelper.image.registry The OCI registry to pull the spiffe helper image from
```